console/console-init/ui/src/modules/endpoints/components/EndPointList/EndPointList.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import {
+  Table,
+  TableHeader,
+  TableVariant,
+  TableBody,
+  IRowData
+} from "@patternfly/react-table";
+import { StyleForTable } from "modules/address-space";
+import { css } from "@patternfly/react-styles";
+import { IEndPoint } from "modules/endpoints/EndpointPage";
+
+interface IEndPointListProps {
+  endpoints: IEndPoint[];
+}
+const EndPointList: React.FunctionComponent<IEndPointListProps> = ({
+  endpoints
+}) => {
+  const tableColumns = ["Name", "Type", "Host", "Ports", ""];
+  const toTableCells = (row: IEndPoint) => {
+    const tableRow: IRowData = {
```

Comment:
I would like to suggest use object destructuring here to avoid calling row object every time.
const {name,type,host, ports}=row;